css/style.css

```diff
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+  padding: 5%;
+  color: #172b4d;
+  background-image: url("/images/header-illsutration-mobile@2x.png");
+}
+
+.header {
+  /* font-family: 'Inter'; */
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+}
+
+.header span {
+  color: green;
+}
+
+.intro {
+  /* font-family: 'Inter'; */
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contacts {
+  display: flex;
+  justify-content: center;
+  align-items: baseline;
+}
+
+i {
+  color: black;
+}
+
+.contacts i {
+  padding-left: 1em;
+  font-size: 25px;
+}
+
+.small {
+  display: none;
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    background-image: url("/images/header-illsutration-mobile@2x.png");
+    background-size: 370px;
+    background-position: 60% 70%;
+  }
+
+  header {
+    display: flex;
+    justify-content: space-between;
```

Comment:
- [ ] From your design, I don't know the template you have chosen from the Figma design to work on but from the project requirement, you must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) but as you can see, your design hee does not match any template
![Screenshot from 2023-01-12 14-39-21](https://user-images.githubusercontent.com/92303891/212097550-bb0bc884-1252-45f7-bd46-03738ae3447a.png)